### ISSUE TITLE
Release 2.5.0

### ISSUE THREAD

**Issue by replay** [grafana/mimir#3525]
This issue is tracking the progress of the release 2.5.0. The release candidate is scheduled for the week of 28th Nov.

### 2.5.0-rc.0

- [x] Write release notes document [PR](https://github.com/grafana/mimir/pull/3580)
- [x] PRs that should be merged before cutting the rc
  - [x] Fix panic in OTLP ingestion [PR](https://github.com/grafana/mimir/pull/3538)
  - [x]  Hide TSDB block ranges period config from doc and mark it experimental [PR](https://github.com/grafana/mimir/pull/3518)
  - [x]  Helm OTLP endpoint missing [PR](https://github.com/grafana/mimir/pull/3479)
  - [x]  Remove "Inflight requests" from object store panels [PR](https://github.com/grafana/mimir/pull/3521)
- [x] PRs to cherry-pick after `2.5.0` was cut from `r214`
  - [x] Remove "Inflight requests" from object store panels [original PR](https://github.com/grafana/mimir/pull/3521) / [backport PR](https://github.com/grafana/mimir/pull/3573)
  - [x] Improve networking panels to work with read write deployment mode [original PR](https://github.com/grafana/mimir/pull/3519) / [backport PR](https://github.com/grafana/mimir/pull/3574)
  - [x] Refactoring: simplified dashboards resources and networking panels utilities [original PR](https://github.com/grafana/mimir/pull/3531) / [backport PR](https://github.com/grafana/mimir/pull/3578)
  - [x] Hide TSDB block ranges period config from doc and mark it experimental [original PR](https://github.com/grafana/mimir/pull/3518) / [backport PR](https://github.com/grafana/mimir/pull/3575)
  - [x] [otlp] Fix panic in dropped count (again!)  [original PR](https://github.com/grafana/mimir/pull/3538) / [backport PR](https://github.com/grafana/mimir/pull/3576)
  - [x] Updated dashboard screenshots [original PR](https://github.com/grafana/mimir/pull/3553/) / [backport PR](https://github.com/grafana/mimir/pull/3577)
  - [x] Fixed Changelog 
    - [x] [original PR1](https://github.com/grafana/mimir/pull/3550) / [backport PR1](https://github.com/grafana/mimir/pull/3579)
    - [x] [original PR2](https://github.com/grafana/mimir/pull/3572) / [backport PR2](https://github.com/grafana/mimir/pull/3581)
- [x] Any feature we can promote to stable?
  - [x] None
- [x] Any deprecated features to drop?
  - [x] None
- [x] Check repo for TODOs that must be addressed in 2.5.0
- [x] Update `CHANGELOG.md`
  - [x] Add any missing CHANGELOG entry [PR](https://github.com/grafana/mimir/pull/3550)
  - [x] Sort CHANGELOG entries
  - [x] Cut the changelog [PR (same as above)](https://github.com/grafana/mimir/pull/3550)
- [x] Notify any open PR to rebase and move the CHANGELOG entry, using `./tools/release/notify-changelog-cut.sh`
- [x] Update dashboard screenshots [PR](https://github.com/grafana/mimir/pull/3553/)
- [x] Create release branch
- [x] Publish the release candidate
  - [x] Update VERSION and remove "main / unreleased" section from the CHANGELOG [PR](https://github.com/grafana/mimir/pull/3583)
  - [x] Tag the release
  - [x] Wait until the CI pipeline succeeds
  - [x] Create a pre-release on GitHub
  - [x] Merge the release branch `release-x.y` into `main`
  - [x] Announce the rc on social media ([twitter](https://twitter.com/r3pl4y/status/1598398260657651712) / [mastodon](https://grafana.social/@replay/109440066614139674))
- [x] Publish the Helm rc
- [x] Vendor the release commit of Mimir into GEM ([PR](https://github.com/grafana/backend-enterprise/pull/3968/))

**Comment by replay** [comment]
In this comment I'm going to track the status of the stable release

- [ ] Publish the Mimir stable release
  - [ ] [Write release notes](https://github.com/grafana/mimir/blob/main/RELEASE.md#write-release-notes-document)
    - Ensure the any change to release notes in `main` has been cherry picked to the release branch
  - [ ] Update version in release-<version> branch
    - VERSION
    - CHANGELOG
    - `operations/mimir/images.libsonnet` (`_images.mimir` and `_images.query_tee` fields)
    - `operations/mimir-rules-action/Dockerfile` (`grafana/mimirtool` image tag)
  - [ ] [Tag the release](https://github.com/grafana/mimir/blob/main/RELEASE.md#how-to-tag-a-release)
    ```bash
    git checkout release-<version>
    ./tools/release/tag-release.sh
    ```
  - [ ] Wait until the CI pipeline succeeds
  - [ ] [Create a release on GitHub](https://github.com/grafana/mimir/blob/main/RELEASE.md#creating-release-on-github)
    ```bash
    git checkout release-<version>
    ./tools/release/create-draft-release.sh
    ```
  - [ ] [Merge the release branch release-<version> into main](https://github.com/grafana/mimir/blob/main/RELEASE.md#merging-release-branch-into-main)
    ```bash
    ./tools/release/create-pr-to-merge-release-branch-to-main.sh
    ```
  - [ ] Announce the release on socials
  - [ ] Open a PR to add the new version to the backward compatibility integration test (`integration/backward_compatibility_test.go`)
  - [ ] [Publish dashboards to grafana.com](https://github.com/grafana/mimir/blob/main/RELEASE.md#publish-a-stable-release)
- [ ] Publish the [Helm release candidate](https://github.com/grafana/mimir/blob/main/RELEASE.md#release-helm)
  - _Ask people working on Helm to do it_
